nzo/url-encryptor-bundle/2020-05-03.yaml

```diff
@@ -4,5 +4,5 @@ cve:       ~
 branches:
     "5.x":
         time:     2020-05-03 16:54:00
-        versions: [<5.0.1]
+        versions: [<4.3.2, 5.0.0]
```

Comment:
Can you create two branches? One for 4.x and one for 5.x?

Reply:
Done.